### PR TITLE
fixed routing and error on login/signup

### PR DESCRIPTION
**Pull request by VL4DYSL4V**

### CODE REVIEW

**Review by VolodymyrYavorskyi — APPROVED** [review]

File: slidez-fe-core/src/App.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { HashRouter, Switch } from 'react-router-dom'
 import PublicRoute from './common/routes/public-route'
 import Dashboard from './pages/dashboard/Dashboard'
-import PrivateRoute from './common/routes/PrivateRoute'
+import PrivateRoute from './common/routes/private-route'
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/App.tsx]: Name files from big letter if we make so with all files

File: slidez-core/src/main/java/com/binarystudio/academy/slidez/SlidezApplication.java [hunk 2]
```diff
@@ -4,8 +4,11 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-@SuppressWarnings({"FinalClassCheck", "HideUtilityClassConstructor"}) class SlidezApplication {
+@SuppressWarnings({ "FinalClassCheck", "HideUtilityClassConstructor" })
+class SlidezApplication {
+
```
Comment (Anatolii-Ivanov) [inline slidez-core/src/main/java/com/binarystudio/academy/slidez/SlidezApplication.java:9]: Don't touch this class, it seems to work now, so please, don't commit it, or Kiril will be angry)

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

File: slidez-fe-core/src/App.tsx (hunk 1 above)
Comment (VL4DYSL4V) [reply slidez-fe-core/src/App.tsx]: Majority of files on frontend named in-this-style, do I have to rename them as well?

File: slidez-core/src/main/java/com/binarystudio/academy/slidez/SlidezApplication.java (hunk 2 above)
Comment (VL4DYSL4V) [reply slidez-core/src/main/java/com/binarystudio/academy/slidez/SlidezApplication.java:9]: In this commit I only ran linter on backend, but I'll remove it

File: slidez-fe-core/src/App.tsx (hunk 1 above)
Comment (Anatolii-Ivanov) [reply slidez-fe-core/src/App.tsx]: I think we name components InThisStyle
Helpers and services can be in-this-style

Comment (Anatolii-Ivanov) [reply slidez-fe-core/src/App.tsx]: Private and Public route are components

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**